core/src/fees.ts

```diff
@@ -49,6 +60,15 @@ export const enrichAuctionWithTransactionFees = async function (
     let combinedSwapFeesETH = fees.swapTransactionFeeETH;
     let combinedBidFeesETH = fees.bidTransactionFeeETH;
 
+    if (auction.marketData && fees.swapTransactionVariableFeeDAI) {
+        const swapTransactionVariableFeePerUnitDAI = fees.swapTransactionVariableFeeDAI.dividedBy(
+            auction.collateralToCoverDebt
+        );
+        Object.values(auction.marketData).forEach(market => {
+            market.marketUnitPrice = market.marketUnitPrice.plus(swapTransactionVariableFeePerUnitDAI);
+        });
```

Comment:
As suggested, please don't modify values inside functions, this will create side effects in javascript. I would say you don't need to modify this function at all, as it is not suitable for modifying values, but for `enriching` objects with extra key-value pairs